### PR TITLE
adding the spec host for ingress in the detail card

### PR DESCRIPTION
**Pull request by Kh40sz3r0**

### CODE REVIEW

File: src/app/frontend/ingress/detail/info.html [hunk 1]
```diff
@@ -24,6 +24,9 @@
     <kd-info-card-entry title="[[Namespace|Resource info details section namespace entry.]]">
       {{::$ctrl.ingress.objectMeta.namespace}}
     </kd-info-card-entry>
+    <kd-info-card-entry title="[[Url Spec|Resource url spec for the ingress.]]" ng-repeat="rule in ::$ctrl.ingress.spec.rules">
```
Comment (floreks) [inline src/app/frontend/ingress/detail/info.html]: Run `gulp build` or `gulp generate-xtbs` to regenerate translations.

Comment (maciaszczykm) [reply src/app/frontend/ingress/detail/info.html]: s/Url/URL

**Comment by maciaszczykm** [comment]
Any progress?

**Comment by maciaszczykm** [comment]
Closing as stale. Please reopen if you are willing to perform requested changes.

**Pull request closed without merging**